Sources/main.cpp

```diff
@@ -19,6 +22,33 @@
 #include "core/FastIsType.h"
 #include "AutogenMetadata.h"
 
+QFile logs("error.log");
+
+void myMessageOutput(QtMsgType type, const QMessageLogContext &context, const QString &msg)
```

Comment:
1. Functions names should camel cased.
2. `my` prefix is bad (it is good to use it in examples, but not in the real code). Better to name it just `KvMessageHandler`.
3. `msg` -> `message`